fiftyone/utils/sam.py

```
@@ -419,6 +439,60 @@ def required_keys(self):
             raise ValueError(f"Undefined required keys for {self.mode.name}")
 
 
+class SegmentAnythingImageGetItemForVideo(SegmentAnythingImageGetItem):
+    """Workaround for applying image model to video reader frames.
+
+    Frames are not stored on disk and therefore cannot be loaded.
+    """
+
+    def __call__(self, d):
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__call__`

(ANN204)